### PR TITLE
fixed merge conflict issue on main branch

### PR DESCRIPTION
**Pull request by john-chamberlin**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [ x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by rafmolina — APPROVED** [review]
conflict to main resolved

File: src/hooks/useMapSearch.js [hunk 1]
```diff
@@ -65,14 +65,11 @@ export default function useMapSearch() {
...
       const start = dates?.[0] ? dates[0] : moment(0);
       const end = dates?.[1] ? dates[1] : moment();
- bugfix/incidents-submit-button
       const matches = dateIncidentList
         .filter(incident => moment(incident.date).isBetween(start, end))
         .map(incident => incident.id);
-=======
 
       //const matches = dateIncidentList.filter(incident => moment(incident.date).isBetween(start, end)).map(incident => incident.id))
- Main
```
Comment (Brandonx1123) [inline src/hooks/useMapSearch.js:75]: awesome

**Review by Brandonx1123 — APPROVED** [review]

**Review by robert-petersen — APPROVED** [review]
nice fix

**Review by rafmolina — APPROVED** [review]
conflict to merge fixex

**Comment by stop0** [comment]
Nice Job.

**Pull request merged**